workflows/create-pull-request.yaml

```diff
@@ -0,0 +1,59 @@
+name: Create Pull request
+
+on:
+  schedule:
+    # Runs "at minute 44 past midnight" (see https://crontab.guru)
+    - cron: "44 0 * * *"
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    env:
+      bot_username: BotName
+      bot_email: BotMail@gmail.com
+    strategy:
+      fail-fast: false
+      matrix:
+        repos: ["de-german", "fr-french"]
```

Comment:
Does it matter in which repository the bot runs? Are there any benefits / drawbacks of running it in the translation repository WRT the origin repository?